lib/ansible/modules/monitoring/grafana_dashboard.py

```diff
@@ -47,10 +47,15 @@
     description:
       - State of the dashboard.
     required: true
-    choices: [ absent, export, present ]
+    choices: [ absent, export, present, list ]
     default: present
+  uid:
```

Comment:
Perhaps move this after `slug` to preservice alphabetical ordering?

Reply:
ok, i've moved uid below slug.